pkg/constants/constants.go

```diff
@@ -32,4 +32,5 @@ const (
 
 	VSphereCredentialsName = "vsphere-credentials"
 	EksaLicenseName        = "eksa-license"
+	EksaPackagesName       = "eksa-packages"
```

Comment:
What is this used for? I see it is passed into the kubectl flag, but it isn't clear how it is different from just packages. If it is a namespace, is there a reason why we aren't putting it under `eksa-system`?

Reply:
We are planning to store all the installed packages under the eksa-packages namespace. The packages we are installing don't qualify as system level packages hence the reason of having a separate namespace.